### DISCUSSION TITLE
Object Oriented Programming

### DISCUSSION THREAD

**Discussion by hakimnazry24** [wilfredinni/python-cheatsheet#160] [Ideas]

**Comment by wilfredinni** [comment]
Hey! It's been in the backlog for a while now, I hope I can find the time soon. PR are welcome too!

**Reply by noraa-july-stoke** [reply]
hey where should i put new work on the OOP basics?, under "cheatsheets"?

**Reply by noraa-july-stoke** [reply]
sorry didnt have time to get around to it right away but was going to start work on it this week.

**Comment by wilfredinni** [comment]
In case anyone was wondering, the OOP section has already been created by @noraa-july-stoke. Therefore, I would like to extend my gratitude and say thank you!
